children/tests/test_api.py

```diff
@@ -593,13 +593,25 @@ def test_get_available_events(snapshot, guardian_api_client):
         relationship__guardian__user=guardian_api_client.user
     )
     variables = {"id": to_global_id("ChildNode", child.id)}
-    occurrences = OccurrenceFactory.create_batch(3, time=timezone.now())
+    # Unpublished occurrences
+    OccurrenceFactory.create(time=timezone.now())
+
+    # Published occurrences
+    occurrence = OccurrenceFactory.create(
+        time=timezone.now(), event=EventFactory(published_at=timezone.now())
```

Comment:
Just a tip in case you didn't know, I think you could do this also like `event__published_at=timezone.now()`